src/ButtonThread.cpp

```diff
@@ -41,8 +40,8 @@ ButtonThread::ButtonThread() : OSThread("Button")
     }
 #elif defined(BUTTON_PIN)
     int pin = config.device.button_gpio ? config.device.button_gpio : BUTTON_PIN; // Resolved button pin
-#if defined(HELTEC_CAPSULE_SENSOR_V3)
-    this->userButton = OneButton(pin, false, false);
+#ifdef BUTTON_ACTIVE_LOW 
```

Comment:
Looks like the HELTEC_CAPSULE_SENSOR_V3 block got accidentally deleted

Reply:
I got ,so sorry about that i ll change it right away.